CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 
 ## Changes since v7.0.1
 
+- [#1064](https://github.com/oauth2-proxy/oauth2-proxy/pull/1064) Add support for setting groups on session when using basic auth
```

Comment:
Be sure to get your name in for attribution

Reply:
Added! might need another tick!